resources/EPvalidator.rb

```diff
@@ -87,6 +83,15 @@ def self.run_validator(hpxml_doc)
         "/HPXML/Building/BuildingDetails/MiscLoads/PlugLoad[PlugLoadType='TV other']" => zero_or_one, # See [Television]
       },
 
+      # [BuildingConstruction]
+      "/HPXML/Building/BuildingDetails/BuildingSummary/BuildingConstruction" => {
+        "NumberofConditionedFloors" => one,
+        "NumberofConditionedFloorsAboveGrade" => one,
+        "NumberofBedrooms" => one,
+        "ConditionedFloorArea" => one,
+        "AverageCeilingHeight | ConditionedBuildingVolume" => one_or_more,
```

Comment:
How about switching the order since `ConditionedBuildingVolume` is the primary input?